cl/search/templates/includes/no_results.html

```diff
@@ -34,7 +34,7 @@ <h2 class="alt">
       {% elif error_message == "unbalanced_quotes" %}
         Did you forget to close one or more quotes?
       {% elif error_message == "disallowed_wildcard_pattern" %}
-        The query contains a disallowed expensive wildcard pattern.
+        The query contains a <a href="{% url "advanced_search" %}#disallowed-expensive-wildcards">disallowed expensive wildcard</a> pattern.
```

Comment:
I don't think most people understand what it means to be an "expensive" wildcard, so, I suggest:

```suggestion
        The query contains a <a href="{% url "advanced_search" %}#disallowed-expensive-wildcards">disallowed wildcard</a> pattern.
```

Reply:
Sure, removed expensive. Also removed from the API error message.